Packs/Gamma/Integrations/Gamma/Gamma.py

```diff
@@ -0,0 +1,321 @@
+
+    :type max_results: int
+    :param max_results: the max number of violations to pull, bound by MAX_INCIDENTS_TO_FETCH
+    """
+
+    try:
+        first_fetch_violation = int(first_fetch_violation)
+        max_results = int(max_results)
+    except:
+        raise ValueError("first_fetch_violation and max_limit must be integers")
+
+    if not first_fetch_violation > 0:
+        raise ValueError("first_fetch_violation must be equal to 1 or higher")
+    if not max_results > 0:
+        max_results = 10
+    elif max_results > MAX_INCIDENTS_TO_FETCH:
+        max_results = MAX_INCIDENTS_TO_FETCH
+
+    # get the last violation id fetched, if exists
+    starting_violation = last_run_violation.get('starting_violation', first_fetch_violation)
+
+    most_recent_violation = starting_violation
+    incidents = []
+    violations = client.get_violation_list(starting_violation, max_results)
+
+    for item in violations['response']:
+        incident_violation = item['violation_id']
+        incident_time_ms = item['violation_event_timestamp'] * 1000
+
+        if incident_violation <= most_recent_violation:
+            continue
+
+        incident = {
+            "name": f'Gamma Violation {incident_violation}',
+            "occurred": timestamp_to_datestring(incident_time_ms),
+            "rawJSON": json.dumps(item)
+        }
+
+        incidents.append(incident)
+
+        # update last run if violation id is greater than last fetch
+        if incident_violation > most_recent_violation:
+            most_recent_violation = incident_violation
+
+    next_run_violation = {'starting_violation': most_recent_violation}
+
+    return next_run_violation, incidents
+
+
+def get_violation_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    :type client: Client
+    :param client: Gamma client
+
+    :param args: all command arguments, usually passed from demisto.args()
+        args['name'] is used as input name
+
+    :return:
+        A CommandResults object that is then passed to return_results
+
+    :rtype: ``CommandResults``
+    """
+
+    minimum_violation = args.get("minimum_violation", 1)
+    limit = args.get("limit", 10)
+
+    if not int(minimum_violation) >= 1:
+        raise ValueError("minimum_violation must be greater than 0")
+    if not int(limit) >= 1 or not int(limit) <= 100:
+        raise ValueError("limit must be between 1 and 100")
+
+    v_list = client.get_violation_list(minimum_violation, limit)
+
+    note = ''
+    if v_list['response'][0]['violation_id'] != int(minimum_violation):
+        note += f'Violation with the minimum_violation ID does not exist. Showing violations pulled from the next available ID: {v_list["response"][0]["violation_id"]} \r'
+
+    human_readable = note
+    for i in v_list['response']:
+        violation_id = i['violation_id']
+        human_readable += f'### Violation {i["violation_id"]} \r' \
```

Comment:
I can suggest looking at the `tableToMarkdown` function in `CommonServerPython`.

Reply:
Got it, we will fix it in the next release